be/src/olap/rowset/rowset.cpp

```diff
@@ -26,6 +26,8 @@
 
 namespace doris {
 
+static bvar::Adder<size_t> g_total_rowset_num("doris_total_rowset_num");
```

Comment:
BE metrics have a rowset_num counter, is it same as bvar::doris_total_rowset_num?
`update_rowsets_and_segments_num_metrics`
![image](https://github.com/apache/doris/assets/13197424/7b4e9491-d548-4475-8dd1-c010f7c84881)